stable/ecr-cleanup/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.1.7"
+appVersion: "0.1.9"
```

Comment:
is there any reason we are skipping `0.1.8`?

Reply:
oops I was changing the wrong version field!